src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -169,6 +208,11 @@ private void Open()
             {
                 File.WriteAllText($"{_chipPath}/export", Convert.ToString(_channel));
             }
+
+            while (!Directory.Exists(_channelPath))
```

Comment:
When we're waiting anyway, we could also wait for permissions.

Reply:
We should unify those permissions utilities from both PRs